_layouts/proposal.html

```diff
@@ -14,6 +19,9 @@ <h1 class="title">{{page.title}}</h1>
     {% endfor %}
     </div>
     {% endunless %}
+    <div style="padding-bottom: 2em;">Tracked by&nbsp;
+      <a href="{{ page.issue }}">{{ page.issue }}</a>
```

Comment:
The first '{{ page.issue }}' has a special char (NBSP) before the 'page' which messes with the jekyll site rendering. So please replace with a regular space.